### ISSUE TITLE
If the default value of a field is a translation string, elasticsearch throws a serialization error.

### ISSUE THREAD

**Issue by Alexgmin** [sabricot/django-elasticsearch-dsl#66]
Models
```python
from django.utils.translation import ugettext_lazy as _
from django.db import models

class Account(models.Model):
    name = models.CharField(
        default=_('Account'),
    )

class Transaction(models.Model):
    dest_account = models.ForeignKey(Account, related_name='in_transactions')
    origin_account = models.ForeignKey(Account, related_name='out_transactions', null=True)
```
Document
```python
class TransactionDocument(DocType):
    dest_account = NullObjectField(properties={
        'name': fields.TextField(),
    })
```
Code
```python
Transaction.objects.create(
    dest_account=Account.objects.create()
)
```

Now, when an account is created using the default value, this happens
```
TypeError at /public/purchases
Unable to serialize u'Initial Account' (type: <class 'django.utils.functional.__proxy__'>)
Traceback:
...
File "/data/projects/project/lib/python2.7/site-packages/elasticsearch_dsl/serializer.py" in default
  11.         return super(AttrJSONSerializer, self).default(data)

File "/data/projects/project/lib/python2.7/site-packages/elasticsearch/serializer.py" in default
  34.         raise TypeError("Unable to serialize %r (type: %s)" % (data, type(data)))
```
Since the string translation hasn't been determined, the type is `django.utils.functional.__proxy__`
https://docs.djangoproject.com/en/dev/ref/unicode/#translated-strings

Adding a condition like this one fixes the problem but i don't know where it should go.
```
from django.utils.functional import Promise
if isinstance(data, Promise):
    return str(data)
```

**Comment by sabricot** [comment]
Thank you for reporting the bug. 
I just fixed it in the 0.4.2.  Can you confirm that it is working for you now ?

**Comment by Alexgmin** [comment]
It's fixed now. Thank you so much!
